docs/api/covidcast-signals/google-symptoms.md

```diff
@@ -1,39 +1,39 @@
 ---
-title: Google Symptoms
+title: Google Search Trends symptoms dataset
 parent: Data Sources and Signals
 grand_parent: COVIDcast Epidata API
 ---
 
 # Google Symptoms
 {: .no_toc}
 
-* **Source name:** `google-symptoms`
+* **Source name:** `google-search-trends-symptoms-dataset`
 * **First issued:** 30 November 2020
 * **Number of data revisions since 19 May 2020:** 0
 * **Date of last change:** Never
 * **Available for:** county, MSA, HRR, state (see [geography coding docs](../covidcast_geography.md))
 * **Time type:** day (see [date format docs](../covidcast_times.md))
-* **License:** [CC BY](../covidcast_licensing.md#creative-commons-attribution)
+* **License:** To download or use the data, you must agree to the Google [Terms of Service](https://policies.google.com/terms)
```

Comment:
@capnrefsmmat please review license change

Reply:
The Google ToS are the terms listed in their GitHub repo for this data, so CC BY was never accurate. @shekelto is a Googler and tells us this is the right licensing to specify.

We do need to update `covidcast_licensing.md` to put this data into the "Republished Data" category, though

Reply:
ok great, thanks for taking a look